examples/all-sky/Makefile

```diff
@@ -57,10 +57,10 @@ tests: rrtmgp_allsky
 	$(RUN_CMD) bash all_tests.sh  
 
 check:
-	$${PYTHON-python} ${RRTMGP_ROOT}/examples/compare-to-reference.py --ref_dir ${RRTMGP_DATA}/examples/all-sky/reference --tst_dir ${RRTMGP_ROOT}/examples/all-sky \
+	$${PYTHON-python} "$(RRTMGP_ROOT)/examples/compare-to-reference.py" --ref_dir "$(RRTMGP_DATA)/examples/all-sky/reference" --tst_dir "$(RRTMGP_ROOT)/examples/all-sky" \
```

Comment:
As far as I remember, `${var}` and `$(var)` mean the same thing for GNU Make. Are these changes decorative or does `mingw-w64-x86_64-make` choke with the current syntax?

Reply:
`$var` and `${var}` are equivalent with the latter being more robust in defining variable boundaries e.g. `echo ${var}foo`.
`$(var)` is used for command execution in your shell. In this case, the usage of `$(RRTMGP_ROOT)` is incorrect and leads to an erroneous expansion of the variable under `msys2` + `mingw-make`.

There are some action runs in my fork where that is demonstrated, (the link to the Action run currently escapes me).